lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -151,23 +151,43 @@ namespace Js
         }
     }
 
-    uint32 ArrayBuffer::GetByteLengthFromVar(ScriptContext* scriptContext, Var length)
+    uint32 ArrayBuffer::GetToIndex(Var value, int32 errorCode, ScriptContext *scriptContext, uint32 MaxAllowedLength, bool checkSameValueZero)
```

Comment:
>:GetToIndex [](start = 23, length = 11)

Do we need to add unit tests for these logic, or are they already covered? What about test262 results?